x-pack/plugins/monitoring/public/components/elasticsearch/ccr/ccr.test.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Ccr } from './ccr';
+
+describe('Ccr', () => {
+  test('that it renders normally', () => {
```

Comment:
By normally, what it means is the expected default rendering of this component. It's not a very good term but it usually means with the expected props provided, this will render